src/data-workspace/data-entry-cell/data-entry-cell.module.css

```diff
@@ -38,10 +45,11 @@
     background: var(--colors-grey300);
 }
 
-.invalid:not(.highlighted) {
+.invalid:not(.active) {
```

Comment:
Using the `highlighted`-selector prevented the highlighted-but-unfocused cell to not show the red-background.